test/test_unit.jl

```diff
@@ -414,7 +414,7 @@ end
     indicator_hg = IndicatorHennemannGassner(1.0, 0.0, true, "variable", "cache")
     @test_nowarn show(stdout, indicator_hg)
 
-    limiter_idp = SubcellLimiterIDP(true, [1], 0.1, "cache")
+    limiter_idp = SubcellLimiterIDP(true, [1], true, [1], 0.1, "cache")
```

Comment:
Didn't you want to disallow non-string arguments?

Reply:
Here, it is only tested that the `show` routine works without warnings and not the constructor itself.
I still save the variables as a vector of `Int`s in the `struct`. So, I have to pass them like I do, right?

I copied this way of testing from other indicators like `IndicatorHennemannGassner`, `Löhner`, `Max`.
In general, this test is not that useful since no real functionality is tested. Maybe we should adapt this in the future :thinking:

Reply:
Ah, ok. Then it's ok I guess.